x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_trends.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient, SearchResponse } from 'kibana/server';
+import { BENCHMARK_SCORE_INDEX_PATTERN } from '../../../common/constants';
+import { Stats } from '../../../common/types';
+import { calculatePostureScore } from './get_stats';
+
+export interface ScoreTrendDoc {
+  '@timestamp': string;
+  total_findings: number;
+  passed_findings: number;
+  failed_findings: number;
+  score_by_cluster_id: Record<
+    string,
+    {
+      total_findings: number;
+      passed_findings: number;
+      failed_findings: number;
+    }
+  >;
+}
+
+export const getTrendsAggsQuery = () => ({
+  index: BENCHMARK_SCORE_INDEX_PATTERN,
+  size: 5,
+  sort: {
+    '@timestamp': {
+      order: 'desc',
+    },
+  },
+});
+
+export type Trends = Array<{
+  timestamp: string;
+  summary: Stats;
+  clusters: Record<string, Stats>;
+}>;
+
+type CorrectSearchResponse<TDocument> = Record<'body', SearchResponse<TDocument>>;
```

Comment:
Looks to be another type error in elastic search. in actuality, query results return nested inside a `body` property, which is not included in the `SearchResult` type, hence the correction.

Reply:
This is not needed, you are getting type errors due to `@ts-ignore` and previous type issues with `sort`.

In addition, you can (and should) remove `meta: true` from the `esClient.search` call, and the results will not be in the `body` property (this is legacy stuff and should not be used)